### ISSUE TITLE
Issue while running the adb-lakehouse example

### ISSUE THREAD

**Issue by suneelsunkara-db** [databricks/terraform-databricks-examples#73]
Thank you so much for building the Terraform registry modules. I am facing some issues with the modules when I use the below terraform.tfvars. I have Contributor rights on the AD App and I am running below command to Auth before running TF Apply. All the resources are getting created but it leaves me with below error messages. Not sure if the deployment has completed or not.

az login --service-principal -u "ae83018d-d8d7-4198-a10f-6187ba1daec1" -p "xyz" --tenant "9f37a392-f0ae-4280-9796-f1864a10effc"

Even though all the resources get deployed in Azure subscription. I still see below errors.

<img width="1025" alt="tfvars cofig used" src="https://github.com/databricks/terraform-databricks-examples/assets/104777003/8f6453e8-17be-41bc-bac1-afd1106ed911">
(url)

[
<img width="1292" alt="error screenshot" src="https://github.com/databricks/terraform-databricks-examples/assets/104777003/d28744b6-ece6-40da-b932-c7caefbcd875">
](url)
Am I missing some configuration? or some Role permissions?
Am I supposed to use a separate storage account name for an external location?

**Comment by bbuxton93** [comment]
I am having a similar issue - did you find a solution?

**Comment by suneelsunkara-db** [comment]
Hi Brian, No. i couldn't find a solution. My guess is that the Azure AD app needs owner permission but not completely sure. Can anyone help here?

**Comment by bbuxton93** [comment]
@suneelsunkara-db I did manage to get it to deploy but continued to have an error with the external data account. I am certainly not a terraform expert, but it seems some of the dependencies may be off and cause the errors. From my testing, the deployment does appear to work across the metastore and other azure resources.

**Comment by suneelsunkara-db** [comment]
@bbuxton93 Yes. same for me. But i can't validate if the metastore is correctly assigned or not. I don't have access to the Account console.
